onbuild/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM finntech/node:6.9.1
+
+# All but package.json is optional
+ONBUILD COPY package.json yarn.lock* .npmrc* npm-shrinkwrap.json* ./
+
+# Install dependencies for native builds
+# This is in one giant command to keep the image size small
+ONBUILD RUN apk add --no-cache --virtual build-dependencies make gcc g++ python git && \
+    npm install --global yarn && \
+    # This script does `yarn install` if a `yarn.lock` file is present, otherwise `npm install`
+    install-dependencies.sh && \
+    npm cache clean && yarn cache clean && \
+    npm uninstall --global yarn && npm uninstall --global npm && \
+    apk del build-dependencies
```

Comment:
>A bundled/transpiled app doesn't have native deps in it either way, so that shouldn't matter

True. Mulling it over a bit more I realise that building isn't a concern, but I think there's reason to question if testing server code outside the image is wise, since there may be native code in play that could behave differently on different architectures.